### ISSUE TITLE
BE_Zip of folder creates incorrect file for Mac

### ISSUE THREAD

**Issue by dansmith65** [GoyaPtyLtd/BaseElements-Plugin#96]
When sending a folder path to `BE_Zip ( file {; archive} )` function's file parameter on Windows, the zip does not extract as expected on a Mac. The opposite works fine (zip a folder on a Mac and that zip file extracts as expected on Windows).

Example windows path: `C:\folder` or `C:\folder\`

If that folder contains the file: 'readme.txt', it will extract like this on a Mac:

- double clicking `folder.zip` creats a folder named `folder`
- in that folder, there will be one file with the name: `folder\readme.txt`

In other words, every file will be renamed to have the folder name prepended to it.

**Comment by nickorr** [comment]
Dan,

Which version does this?  We’ve done some fixes for empty folders lately, that this may be related to.

Cheers,
Nick

**Comment by dansmith65** [comment]
BE_VersionAutoUpdate = 03009007
Preferences > Plug-Ins tab shows it as 3.1a3

**Comment by nickorr** [comment]
Dan,

Another followup.  I think this is resolved now too, let me know if it's not.

Cheers,
Nick
